### PR TITLE
Merge remote-tracking branch 'origin/97019-Content-Changes' into feat…

### PR DESCRIPTION
**Pull request by MosesOkoruwaDfE**
…ure/100585-transfers-content-advisory-board

### Checklist

- [x] Rebased master
- [ ] Cleaned commit history
- [ ] Tested by running locally

### CODE REVIEW

File: Frontend/Pages/Home/Index.cshtml [hunk 1]
```diff
@@ -29,9 +29,6 @@
         <h2 class="govuk-heading-l">
             Projects
         </h2>
-        <h3 class="govuk-heading-m">
```
Comment (jordan-beard) [inline Frontend/Pages/Home/Index.cshtml:32]: Did you meant to remove this h3?

File: Frontend/Pages/Projects/TransferDates/AdvisoryBoard.cshtml.cs [hunk 2]
```diff
@@ -65,8 +66,18 @@ public async Task<IActionResult> OnPostAsync()
                 return Page();
             }
 
-            projectResult.Dates.Htb = AdvisoryBoardViewModel.AdvisoryBoardDate.DateInputAsString();
+            projectResult.Dates.Htb = AdvisoryBoardViewModel.AdvisoryBoardDate.DateInputAsString();
```
Comment (jordan-beard) [inline Frontend/Pages/Projects/TransferDates/AdvisoryBoard.cshtml.cs:69]: Whitespace

File: Frontend/Pages/Projects/TransferDates/AdvisoryBoard.cshtml.cs [hunk 3]
```diff
@@ -65,8 +66,18 @@ public async Task<IActionResult> OnPostAsync()
                 return Page();
             }
 
-            projectResult.Dates.Htb = AdvisoryBoardViewModel.AdvisoryBoardDate.DateInputAsString();
+            projectResult.Dates.Htb = AdvisoryBoardViewModel.AdvisoryBoardDate.DateInputAsString();            
             projectResult.Dates.HasHtbDate = !AdvisoryBoardViewModel.AdvisoryBoardDate.UnknownDate;
+            projectResult.Dates.HtbInserted = DateTime.Now.ToString("dd/MM/yyyy");
+            if(!string.IsNullOrEmpty(projectResult.Dates.HtbInserted))
```
Comment (jordan-beard) [inline Frontend/Pages/Projects/TransferDates/AdvisoryBoard.cshtml.cs:72]: To save space you could use the ternary operator
`projectResult.Dates.HasHtbInserted = !string.IsNullOrEmpty(projectResult.Dates.HtbInserted) 
? true
: false`

**Review by jordan-beard — CHANGES_REQUESTED** [review]
A few small things.

**Review by jordan-beard — APPROVED** [review]

**Review by jordan-beard — APPROVED** [review]

**Review by MartinWheelerMT — APPROVED** [review]

**Pull request closed without merging**